components/supervisor-api/typescript/scripts/publish.js

```diff
@@ -0,0 +1,42 @@
+
+/**
+ * Copyright (c) 2020 TypeFox GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const child_process = require('child_process');
+const qualifier = process.argv[2];
+
+const rootDir = process.cwd();
+const pckDir = path.join(rootDir, "components-supervisor-api-typescript--lib/package");
```

Comment:
It would be nice to reuse this script between components but not sure how. cc @csweichel ?